WordPress/src/main/AndroidManifest.xml

```diff
@@ -107,6 +107,11 @@
         </activity>
 
         <activity
+            android:name=".ui.accounts.LoginActivity"
```

Comment:
When we drop the feature flag, will we drop the `SignInActivity` as well? If not I think we should make that activity name more explicit.